domain/build.gradle.kts

```
@@ -0,0 +1,9 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
```

순수 코틀린모듈이면 아래처럼도 가능합니다! 😄 
```suggestion
    id("kotlin")
```